classes/class-wc-connect-order-item-meta.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+if ( ! class_exists( 'WC_Connect_Order_Item_Meta' ) ) {
+	class WC_Connect_Order_Item_Meta {
```

Comment:
Hold on. Do we really want to do this? Wipe all existing order packaging meta-data and labels history when the plugin is uninstalled? Seems too destructive to me.